### DISCUSSION TITLE
Theme rules (help)

### DISCUSSION THREAD

**Discussion by moiatto** [mapsforge/vtm#1129] [Q&A] [answered]
How can I make only one central icon appear in this area? (same for text?)
because I see several icons and texts scattered around the area.
```
<m k="leisure" v="golf_course" zoom-min="11">
      <area use="golf_course" />
      <m zoom-min="14">
          <symbol src="assets:symbols/sport/golf.svg" />
          <line use="park" />
          <text use="park" />
      </m>
</m>
```

**Comment by devemux86** [comment]
Can try the "area-size" parameter in the theme.

And the Parameters.POLY_LABEL option in the app.

**Comment by moiatto** [comment]
@devemux86 
I have no idea where to start...I'm groping in the dark 😅

can you be a little more detailed?

**Comment by devemux86** [comment, marked as answer]
Try to enable the `Parameters.POLY_CENTROID` (documentation [here](https://github.com/mapsforge/vtm/blob/master/vtm/src/org/oscim/utils/Parameters.java#L61-L73)).

Also "area-size" parameter can be used to limit the labels (documentation [here](https://github.com/mapsforge/vtm/blob/master/resources/rendertheme.xsd#L157-L158)).
See in the default theme [here](https://github.com/mapsforge/vtm/blob/master/vtm-themes/resources/assets/vtm/default.xml#L466) how the "area-size" parameter is used for lakes.

**Reply by moiatto** [reply]
Thanks @devemux86.

I will study it.
